app/create_app.py

```diff
@@ -211,6 +211,16 @@ def inject_content_urls():
             feedback_url=url_for("content_routes.feedback"),
         )
 
+    @flask_app.after_request
+    def after_request(response):
+        if "Cache-Control" not in response.headers:
+            response.headers[
+                "Cache-Control"
+            ] = "no-cache, no-store, must-revalidate"
+            response.headers["Pragma"] = "no-cache"
+            response.headers["Expires"] = "0"
+        return response
```

Comment:
IIRC didn't we recently remove this?  Or was that for assessment 🤔

Reply:
cache-control was not implemented in front-end before

Reply:
Ah so it was assessment! thanks